packages/angular/cli/commands/update-impl.ts

```diff
@@ -163,10 +163,12 @@ export class UpdateCommand extends Command<UpdateCommandSchema> {
 
     migrations.sort((a, b) => semver.compare(a.version, b.version) || a.name.localeCompare(b.name));
 
+    this.logger.info(
+      colors.cyan(`** Executing migrations of package '${packageName}' **\n`),
+    );
+
     for (const migration of migrations) {
-      this.logger.info(
-        `** Executing migrations for version ${migration.version} of package '${packageName}' **`,
-      );
+      this.logger.info(`${colors.symbols.pointer}  ${migration.description.replace(/\. /g, '.\n   ')}`);
```

Comment:
As a future addition, I think it would be useful to have a `title` field in the migration definition.  So that the description doesn't need to be manipulated.

Reply:
The description is also manipulated so that the description itself is not printed in a single line.